platforms/r3-corda-ent/charts/generate-pki/README.md

```diff
@@ -3,4 +3,188 @@
 [//]: # (SPDX-License-Identifier: Apache-2.0)
 [//]: # (##############################################################################################)
 
-chart for creating initial certificates
\ No newline at end of file
+<a name = "deploy generate-pki"></a>
+# Generate-pki Deployment
+
+- [Generate-pki Deployment Helm Chart](#Generate-pki-deployment-helm-chart)
+- [Prerequisites](#prerequisites)
+- [Chart Structure](#chart-structure)
+- [Configuration](#configuration)
+- [Deployment](#deployment)
+- [Contributing](#contributing)
+- [License](#license)
+
+<a name = "Generate-pki-deployment-helm-chart"></a>
+## Generate-pki Deployment Helm Chart
+---
+This [Helm chart](https://github.com/hyperledger/bevel/tree/main/platforms/r3-corda-ent/charts/generate-pki) generates the certificates and configure a PKI for a CENM application. 
```

Comment:
deploys the pki to generate the corda complaint certificate hierarchy for the CENM services